lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -60,6 +64,12 @@ function runA11yChecks() {
         node.path = getNodePath(node.element);
         // @ts-ignore - getOuterHTMLSnippet put into scope via stringification
         node.snippet = getOuterHTMLSnippet(node.element);
+
+        const rect = node.element.getBoundingClientRect();
+        if (rect.width > 0 && rect.height > 0) {
+          node.boundingRect = JSON.parse(JSON.stringify(rect));
```

Comment:
Why do we need to do the JSON stringify/parse here?

Reply:
It's because the properties on DOMRect are getters, and the protocol doesn't serialize getters. I'll add a comment.